.github/workflows/test-build-deploy.yml

```diff
@@ -104,68 +104,37 @@ jobs:
         run: |
           docker run -v ${PWD}/docs/sources:/hugo/content/docs/mimir/latest -e HUGO_REFLINKSERRORLEVEL=ERROR --rm grafana/docs-base:latest /bin/bash -c 'make hugo'
 
-  build-mimir:
-    runs-on: ubuntu-20.04
+  build:
```

Comment:
We had two separate jobs (build-mimir and build-tools) to speed up CI. Other CI jobs depend on build-mimir but not build-tools. How longer does CI take, getting back to a single build job?

Reply:
I put it together during this experiment. I can split it again now that it all works. `build` takes about 4 minutes. Let's see splitting the two will change it.

Reply:
Multiarch build-mimir and build-tools take about 3-4 minutes, so benefit of splitting seems small compared to combined step, which takes ±4mins. I will use combined `build` step only.